_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -28,7 +28,7 @@ The rule applies to any [keyboard event][]:
 For each test target:
  - an [instrument][] to prevent any result of the [keyboard event][] is available; or
```

Comment:
If an event has more than one outcome but these are disabled by different instruments, it will fail the rule but pass the SC.
(e.g. `+` both adds an item to the list and jumps focus to the list, a button stops `+` from jumping, another one stops it from adding elements, both buttons can be activated at the same time)